docs/lint_errors.md

```diff
@@ -120,6 +120,12 @@ The following variables are depreciated and fail the test if they are still pres
 
 nf-core pipelines must have CI testing with GitHub actions or Travis.
 
+There are three main CI test files
```

Comment:
I think we should merge this section with `For GitHub actions CI workflow`  and then maybe create distinct headings for `GitHub Actions` and `Travis`